docs/uab_cloud/tutorial/volumes.md

```diff
@@ -198,3 +198,30 @@ Click "Delete Volume" again to delete the volume permanently.
 
     It will not be possible to delete a volume if it has an associated [volume snapshot](../snapshots.md). The snapshot will need to be deleted first.
 <!-- markdownlint-enable MD046 -->
+
+## Increasing a Persistent Volume
```

Comment:
Note the previous section is not part of the tutorial. I also like to put tutorials in order of the operations that would be performed, like a sort of timeline. So, creation/acquisition always comes first, and deletion/relinquishing always comes last.

In short, let's move this before deletion.